source/gloperate/include/gloperate/rendering/CameraUtils.h

```diff
@@ -0,0 +1,41 @@
+
+#pragma once
+
+
+#include <glm/mat4x4.hpp>
+
+#include <gloperate/gloperate_api.h>
+
+
+namespace gloperate
+{
+
+/**
+*  @brief
+*    Compute orthographic projection matrix from perspective parameters
+*
+*  @param[in] fovy, aspectRatio, zNear, zFar
+*    same as for glm::perspective
+*
```

Comment:
As can be seen throughout the whole project, please add no lines between several parameters